File: Python3/Medium/Accepted/bulls-and-cows_5.py
class Solution:
    def getHint(self, secret: str, guess: str) -> str:
        
        bulls, cows = 0, 0
        secret_list = list(secret)
        guess_list = list(guess)
        for i in range(len(secret_list)):
            if secret_list[i] == guess_list[i]:
                bulls += 1
                secret_list[i] = guess_list[i] = 'B'
        for i in range(len(secret_list)):
            if secret_list[i] == 'B':
                continue
            if secret_list[i] in guess_list:
                cows += 1
                guess_list[guess_list.index(secret_list[i])] = 'C'
        return str(bulls) + 'A' + str(cows) + 'B'

File: Java/Medium/CompileError/bulls-and-cows_5.java
class Solution {
    public String getHint(String secret, String guess) {
        
        int[] secretDigits = new int[10];
        int[] guessDigits = new int[10];
        int bulls = 0;
        int cows = 0;
        for (int i = 0; i < secret.length(); i++) {
            int secretDigit = secret.charAt(i) - '0';
            int guessDigit = guess.charAt(i) - '0';
            if (secretDigit == guessDigit) {
                bulls++;
            } else {
                secretDigits[secretDigit]++;
                guessDigits[guessDigit]++;
            }
        }
        for (int i = 0; i < 10; i++) {
            cows += Math.min(secretDigits[i], guessDigits[i]);
        }
        }
        return bulls + "A" + (secret.length() - bulls - cows) + "B";
    }
}